lib/pltr/v2/reducers/lines.js

```diff
@@ -43,7 +43,7 @@ export default function lines(state = initialState, action) {
         {
           id: nextId(state),
           bookId: actionBookId,
-          title: '',
+          title: i18n('Main Plot'),
```

Comment:
I don't think we want to change the reducer to always add this title.

Look in `project/BookList` where books get added. I think the solution lies there. And that might be what you meant when you were asking about `addLineWithValues`. I think you'll need to use that action